brom.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+python main.py

```

Comment:
it doesnt run  and gives error ! 
while all drivers is good

Reply:
It works for everyone, but you don't, very strange